rfcs/test_scoring_change.md

```diff
@@ -0,0 +1,61 @@
+# RFC: Change results aggregation on wpt.fyi and visualization on Interop-20** views
+
+## Summary
+
+The aggregate scoring that is displayed in the results view of wpt.fyi is
+inaccurate in some aspects. This proposal will describe a change to the way
+that test results are aggregated, as well as a change to how those results are
+displayed for Interop 2022 scoring purposes.
+
+## Details
+There are a number of aspects to this change. Each subsection will describe
+these aspects in detail.
+### 1. Remove harness status from subtest aggregation
+
+When interpreting the results of test suite runs, a status is present that
+represents either the overall test's status or the "harness status", which
+represents whether any harness errors were present for `testharness.js` test
+runs. The current results aggregation method counts this harness status as a
+separate subtest itself, which can inflate the overall passing percentage of
+a test. There are many instances of a test outright failing its only subtest,
+but the test is weighted as 50% passing because there were no harness errors
+[(example)](https://wpt.fyi/results/audio-output/selectAudioOutput-sans-user-activation.https.html?run_id=5642791446118400&run_id=5668568732532736&run_id=5739999172493312&run_id=5735075831349248).
+The solution proposed is to no longer aggregate this harness status into the
+subtest totals.
+
+### 2. Display additional information if a harness error exists.
+
+Although it seems intuitive to simply disregard the harness status when
+calculating tests, there are instances where a harness error will occur, which
+causes a number of subtests to not be executed. In some circumstances, it is
+possible that a harness error will occur, allowing for some subtests to run
+and pass as expected, but some subtests to be not run and not be registered,
+as they are not present during aggregation. This has the possibility of hiding
+the scope of failure for a specific test in this scenario. The solution
+proposed is to mark these tests with harness errors with some additional
+visual notice that this a harness error occurred. This is to signify that
+the results should not be taken at face value and engineers should investigate
+the test further.
+
+### 3. Interop-20** scores will display more accurate scores.
```

Comment:
Can you add some details about how this will be done, with the query parameter?

Reply:
Added 👌

Reply:
Can we change the wording here away from "more accurate"? Could just be `Implement optional display of Interop-20xx scores for labelled tests`

Reply:
I've made that wording change.

Reply:
Thanks @foolip!